config/routes.rb

```diff
@@ -65,44 +69,52 @@
   #merchant
   namespace :merchant do
     get '/dashboard', to: 'dashboard#show'
-    get '/items', to: 'items#index'
-    patch '/items/:id', to: 'items#update'
-    delete '/items/:id', to: 'items#destroy'
-    get '/items/new', to: 'items#new'
-    post '/items', to: 'items#create'
-    get 'items/:id/edit', to: 'items#edit'
-    get '/orders/:id', to: 'orders#show'
-    patch '/item_orders/:id/', to: 'item_orders#update'
+    resources :items
+    # get '/items', to: 'items#index'
+    # patch '/items/:id', to: 'items#update'
+    # delete '/items/:id', to: 'items#destroy'
+    # get '/items/new', to: 'items#new'
+    # post '/items', to: 'items#create'
+    # get 'items/:id/edit', to: 'items#edit'
+    resources :orders, only: [:show]
+    resources :item_orders, only: [:update]
+    # get '/orders/:id', to: 'orders#show'
+    # patch '/item_orders/:id/', to: 'item_orders#update'
     resources :discounts
   end
 
   #admin
   namespace :admin do
     get "/dashboard", to: "dashboard#show"
-    get '/users', to: 'users#index'
-    get "/users/:user_id", to: "users#show"
-    get '/users/:user_id/password/edit', to: 'users_password#edit'
-    patch '/users/:user_id/password/edit', to: 'users_password#update'
-    get '/users/:user_id/edit', to: 'users#edit'
-    patch '/users/:user_id', to: 'users#update'
-    get '/users/:user_id/orders', to: 'users_orders#index'
-    get '/merchants', to: 'merchants#index'
+    resources :users do
+      resources :orders, only: [:index], controller: 'users_orders'
+      get '/password/edit', to: 'users_password#edit'
+      patch '/password/edit', to: 'users_password#update'     
```

Comment:
couldn't figure out how to remove the password id that comes by default with edit